CHANGES.rst

```diff
@@ -1,31 +1,103 @@
-Upcoming version
-================
+
+Version 0.14.0 (2024-05-21)
+===========================
 Main items of this new release:
```

Comment:
While I don't like advertising my own work, one thing I would like to 'advertise' in this main-items section is the addition of the default queue method combined with the file-input; the consequence of this is that it should be a lot easier to get started with the PyMeasure interface. So something along the lines of:
```
- Add default :code:`queue` method and a :code:`FileInputWidget`, allowing to more quicklly get started with the PyMeasure user interface (:code:`ManagedWindow`).
```
I'll leave it up to you if you agree with this and want to add this to the "Main items" section.

(In Dutch, we have a nice saying about this, which translates to "We from Toilet-Duck recommend Toilet-Duck", but I don't thinkt there is a good English equivalent to this saying).

Reply:
It's ok to add important parts.

Reply:
Thanks for adding your thought regarding important changes.